frontend/src/components/Register.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+const registerStyle = {
+  width: "10%",
+  border: "1px solid black",
+  fontSize: "20px",
+  backgroundColor: "green"
+};
+
+class RegistrationPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: "",
+      password1: "",
+      password2: ""
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    let target = event.target;
+    let value = target.type === "checkbox" ? target.checked : target.value;
```

Comment:
Do we have any checkboxes in the register page? If not, I don't think this ternary operation needs to happen.